pages/widerruf/frist.js

```diff
@@ -0,0 +1,141 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function FristWiderruf() {
+  return (
+    <>
+      <StyledBody>
+        <StyledBreadcrumb>
+          <StyledLink href="/widerruf">/Widerruf und Retoure</StyledLink>/Wie
+          lange kann ich widerrufen?
+        </StyledBreadcrumb>
+        <StyledArticle>
+          <Styledh3>Wie lange kann ich widerrufen?</Styledh3>
+          {`
+    Der Widerruf muss binnen 14 Tagen nach Erhalt der Ware erklärt werden. Das bedeutet nicht, dass binnen dieser Zeit auch die Ware zurück an
+    den Verkäufer gesendet werden muss. Die Ware muss innerhalb weiterer 14 Tage ab Erklärung des Widerrufs
+    versendet werden. 
+    Achte auf die Widerrufsregelungen im Shop des Verkäufers. Dort ist
+    meist der Zeitraum für die Rücksendung der Ware explizit geregelt.
+    Meistens liegt dieser bei 30 Tagen.
+
+    TLDR: binnen 14 Tagen erklären, je nach Widerrufsregelung des
+    Verkäufers zwischen 14 und 30 Tagen ab Erklärung zurück senden
+      `}
+        </StyledArticle>
```

Comment:
It would be better to group your content in paragraphs (`<p>`)

Reply:
ok :-)